sim/pokemon.ts

```diff
@@ -562,7 +562,7 @@ export class Pokemon {
 			stat = this.battle.runEvent('Modify' + statTable[statName], this, null, null, stat);
 		}
 
-		if (statName === 'spe' && stat > 10000) stat = 10000;
+		if (statName === 'spe' && stat > 10000 && !this.battle.format.id.includes('customgame')) stat = 10000;
```

Comment:
Pretty sure this should be ``!this.debugMode``.

Reply:
Well, `!this.battle.debugMode`, but yeah I get your point.